scripts/sensorshow

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+
+"""
```

Comment:
Perhaps a somewhat minor nit, but let's pick either `"` or `'` and be consistent throughout the file. My preference is for `'`